app/controllers/Checkout.scala

```diff
@@ -268,6 +270,20 @@ object Checkout extends Controller with LazyLogging with CatalogProvider {
         _ + _
       }
 
+      val acquisitionData = request.session.get("acquisitionData")
+      if (acquisitionData.isEmpty) {
+        logger.warn(s"No acquisitionData in session")
+      }
+
+      val promotion = subscribeRequest.genericData.promoCode.map(_.get).flatMap(code => tpBackend.promoService.findPromotion(NormalisedPromoCode.safeFromString(code)))
+
+      OphanService(isTestService = tpBackend.environmentName != "PROD")
+        .submit(SubscriptionAcquisitionComponents(subscribeRequest, promotion, acquisitionData))
```

Comment:
is this blocking? or does it return a future? we should probably Future.sequence it into the response if so.

Reply:
it returns an EitherT of a Future, but it's fire & forget so I don't think we need to wait for it